Nehal/output_to_csv_file.ipynb

```diff
@@ -0,0 +1,494 @@
+{
```

Comment:
Is that the running code or the snippet. As i remember that you were using .py file